### ISSUE TITLE
[Issue]: Indexing movies takes hours and does not complete

### ISSUE THREAD

**Issue by deepzeafish** [jellyfin/jellyfin#6975]
### Please describe your bug

Indexing around 180 movies takes hours on a Raspberry Pi 3. I cancelled the indexing process after a few hours as it seemed to be stuck at 80% and rescheduled it with the same result - it got stuck again at about 80%. I tried it again on the next day - same result.
I haven't done any modifications to the OS or installed any plugins or whatsoever for Jellyfin. It's a freshly installed Raspberry Pi.
The logs don't show any errors.

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS: Raspberry Pi OS Lite 5.10
- Virtualization: No
- Clients: Any
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by crobibero** [comment]
What is your file structure?

**Comment by deepzeafish** [comment]
> What is your file structure?

Plain. No subfolders. All movies are in a single folder.

**Comment by crobibero** [comment]
Duplicate of https://github.com/jellyfin/jellyfin/issues/6906
There is a PR that should fix this for 10.8.
